go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/terraform-providers/terraform-provider-datadog
 
 require (
-	github.com/DataDog/datadog-api-client-go/v2 v2.5.1-0.20221214110216-1f5a9877e971
+	github.com/DataDog/datadog-api-client-go/v2 v2.6.1
```

Comment:
Why are we updating the api client? Is this perhaps why the tests are failing?

Reply:
Yes and I needed the latest release with the datadog-api-spec changes which expose the rego policy and other useful fields.